### PR TITLE
[WIP] Website bridge update

### PR DESCRIPTION
**Pull request by dgca**
### What changed?

- Updates home page to include bridge info
- Updates Node App page to include bridge and Ledger info
- Adds OreoWallet and IF Labs to Ecosystem page
- Moves About Us content to Foundation page

Fixes IFL-2906

---

<sub>**Reminder:** If content (docs, blogs, pages) is moved or renamed, please ensure there are no broken links.</sub>

### CODE REVIEW

File: content/ecosystem/ecosystem.ts [hunk 1]
```diff
@@ -16,6 +17,18 @@ export const ECOSYSTEM: Array<{
   image: string;
   highlight?: string;
 }> = [
+  {
+    type: "Apps",
```
Comment (patnir) [inline content/ecosystem/ecosystem.ts:21]:
```suggestion
    type: "Wallets",
```

File: shared/navLinks/links.ts [hunk 2]
```diff
@@ -98,26 +98,20 @@ export const links = [
     ],
   },
   {
-    label: messages.company,
-    color: "pink",
+    label: messages.developers,
```
Comment (cgjohn) [inline shared/navLinks/links.ts:101]: This appears twice in the navigation bar - did we intentionally want to get rid of the company section here?

File: components/BridgeDropdown/BridgeDropdown.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
+import { ReactNode } from "react";
+import { Box, BoxProps, Button, Flex, Text } from "@chakra-ui/react";
+import { useToggle } from "usehooks-ts";
+import { IoChevronDown, IoChevronUp } from "react-icons/io5";
+import { FancyArrowUpRight } from "@/lib/ui";
+
+export function BridgeDropdown() {
+  return (
+    <Box position="relative">
+      <ButtonContents hidden />
```
Comment (cgjohn) [inline components/BridgeDropdown/BridgeDropdown.tsx:10]: Curious why the layering of components is needed here?

File: components/HomePage/Safety/Safety.tsx [hunk 4]
```diff
@@ -0,0 +1,321 @@
...
+      color={color}
+    >
+      <Text textStyle="sm">{children}</Text>
+      <Box transform="scale(0.7)">
+        <FancyArrowRight />
+      </Box>
+      <Box
+        w="0%"
+        h="2px"
+        bg="currentColor"
```
Comment (cgjohn) [inline components/HomePage/Safety/Safety.tsx:247]: TIL currentcolor

File: components/HomePage/WhyIronFish/WhyIronFish.tsx [hunk 5]
```diff
@@ -1,32 +1,77 @@
...
+    description: (
+      <Grid gap={4}>
+        <Text>
+          Every transfer in and out of Iron Fish passes through ChainPort&apos;s
+          sanction screening and real-time threat detection models. We take a
+          strong stance that Iron Fish is to be used for good, and malicious
+          addresses attempting to bridge to Iron Fish will have their funds
+          frozen.
+        </Text>
+        <Flex mt="">
```
Comment (cgjohn) [inline components/HomePage/WhyIronFish/WhyIronFish.tsx:41]: super small nit: we can get rid of the `mt`

Comment (cgjohn) [reply components/HomePage/WhyIronFish/WhyIronFish.tsx:41]: And probably this `<Flex>` all together if we add a `w="fit-content"` to the child

**Review by cgjohn — APPROVED** [review]

**Pull request merged**